src/main/java/aeneas/models/ReleaseBoard.java

```diff
@@ -2,17 +2,35 @@
 
 import java.util.ArrayList;
 
+import javafx.scene.paint.Color;
+
 /**
  * 
  * @author Joseph Martin
+ * @author Logan Tutt
  */
 public class ReleaseBoard extends Board implements java.io.Serializable {
+
   ArrayList<ReleaseNumber> numbers;
 
-  public ReleaseBoard(ArrayList<ReleaseNumber> numbers) {
-    this.numbers = numbers;
+  public ReleaseBoard(ArrayList<ReleaseNumber> numbers){
+    if(numbers == null)
+      this.numbers = new ArrayList<ReleaseNumber>();
+    else
+      this.numbers = numbers;
+
   }
-  
+
+  @Override
+  public Square[][] getSquares(){
+    Square[][] squares = super.getSquares();
+    for(ReleaseNumber num: numbers){
+      if(squares[num.getCol()][num.getRow()] != null && getPieceAtLocation(num.getRow(), num.getCol()) == null)
```

Comment:
getPieceAtLocation is the only intersection method currently for checking if a square is covered by a piece